src/js/stanford_basic.behavior.js

```diff
@@ -25,6 +25,15 @@ export default {
       if (!$sn) {
         $('.su-skipnav--secondary', context).remove();
       }
+
+      // Check for search box and move the second block to the mobile navigation.
+      // Hide it and then only show for mobile sites.
+      var $search = $('.su-masthead .su-site-search', context).length;
+      if ($search) {
+        var $ms = $('.su-masthead nav + .su-site-search', context);
+        $($ms).prependTo('.su-masthead .su-multi-menu > ul', context).wrap('<li></li>');
+      }
```

Comment:
```suggestion
      var $search = $('.su-masthead .su-site-search', context);
      if ($search.length) {
        var $searchcopy = $search.clone();
        $searchcopy.prependTo('.su-masthead .su-multi-menu > ul', context).wrap('<li></li>');
      }
```

Instead of adding a second search block to the same region how about just copying the first one that is there?

Reply:
Oh, IDs become an issue.

Reply:
Can't do that unless we fix the IDs as well. Drat.